examples/run_example.sh

```diff
@@ -1,17 +1,18 @@
 #!/usr/bin/env bash
```

Comment:
IMO for such simple script we should use echo instead of stage and remove all that jazz.
BTW this script is meant to be executed from `examples` directory. Otherwise it will spam logs from all unit/integration tests.

Reply:
Done